### ISSUE TITLE
docs mislead gdrive users to use poor scope

### ISSUE THREAD

**Issue by philpennock** [rclone/rclone#6871]
#### What problem are you are trying to solve?

The docs are recommending a dangerous approach which used to be all that worked with GDrive, but which lead to chaos when people switch to a dedicated `client_id` for syncing _to_ GDrive.

At <https://rclone.org/drive/#drive-file> we see:

> **drive.file**
> With this scope rclone can read/view/modify only those files and folders it creates.
> So if you uploaded files to drive via the web interface (or any other means) they will not be visible to rclone.
> This can be useful if you are using rclone to backup data and you want to be sure confidential data on your drive is not visible to rclone.
> Files created with this scope are visible in the web interface.

But if someone is copying to GDrive and starts with the default `client_id`, and later generates their own `client_id`, they'll suddenly re-upload duplicates of everything, creating a mess.  This was discussed today in a non-rclone community Slack where I tracked down someone's problem to be "almost certainly caused by this".  After all, how does gdrive/rclone know which are the files created by rclone only?  Well, the client app ID seems like the most likely identifier.

#### How do you think rclone should be changed to solve that?

Divide advice into two:
 1. Those using Google Workspace and having access to commercial features
 2. Everyone else

**Google Workspace**

Modern GDrive supports having multiple drives, each of which can have its own permissions, if and only if you have Google Workspace.

So instead of recommending that backups use `scope = drive.file` and having backups go haywire when someone switches app ID, instead recommend that these users create a dedicated shared drive and sync with `scope = drive`

**End-user accounts**

Here rclone should probably recommend setting the `root_folder_id` field and `scope = drive.file`.  If there are subtleties to `root_folder_id` which make this less safe than using `scope = drive.file` then I think there simply are no good choices left, other than to include warnings that this will go wrong when a dedicated `client_id` is created.

<details><summary>keyword grep-bait</summary>
google, google suite, google apps, google-drive, gdrive, sync to gdrive, backups, `type = drive`, duplicated files
</details>

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by animosity22** [comment]
> But if someone is copying to GDrive and starts with the default `client_id`, and later generates their own `client_id`, they'll suddenly re-upload duplicates of everything, creating a mess.

That seems wrong as the Client ID is just the API and changing who is consuming the API should not matter. I've tested that one quite a bit and never have seen that. Love to see a log file or a recreation of that. Minus shared with me, I've never had rclone create a duplicate in quite a bit of testing.

For any changes, feel free to submit a PR to make any adjustments.

**Comment by animosity22** [comment]
> We should probably be more specific about the fact that `drive.file` is tied to a specific `client_id` and any `client_id` can be revoked by google at any moment leaving your data inaccessible.

If both my own personal client ID and rclone client ID all got revoked, I can download via the WebUI to retrieve my data in that scenario.

**Comment by ncw** [comment]
> > We should probably be more specific about the fact that `drive.file` is tied to a specific `client_id` and any `client_id` can be revoked by google at any moment leaving your data inaccessible.
> 
> If both my own personal client ID and rclone client ID all got revoked, I can download via the WebUI to retrieve my data in that scenario.

That is true, and you could rescope your oauth easily enough. I guess I was thinking of `drive.appfolder`

So

> We should probably be more specific about the fact that drive.file is tied to a specific client_id ~~and any client_id can be revoked by google at any moment leaving your data inaccessible~~.
